tests/integration/service/test_integration_files_service.py

```diff
@@ -11,11 +11,6 @@
 from deepset_cloud_sdk._service.files_service import DeepsetCloudFile, FilesService
 
 
-@pytest.fixture
```

Comment:
why can we drop this? Cause we're using the same workspace for both?

Reply:
yes